eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -42,6 +44,44 @@ stages:
                         ReleaseSha: $(Build.SourceVersion)
                         RepoId: Azure/azure-sdk-for-python
                         WorkingDirectory: $(System.DefaultWorkingDirectory)
+                        
+          - deployment: PublishDocs
+            displayName: "Docs.MS Release"
+            condition: ne(variables['Skip.PublishDocs'], 'true')
+            environment: githubio
```

Comment:
Is this the correct environment?

Reply:
Honestly, the `environments` are pointless. You can't individually approve them anyway. I deliberately left this alone given that these are both `docs` related.

Reply:
@mitchdenny Should we consider an enviroment per release pipeline? So that we only have one approval per release?